bible

```diff
@@ -82,7 +82,21 @@ def get_passages(soup):
     text = ""
     for p in passages:
         text += get_passage(p) + "\n\n"
-    return text
+    
+    chars = list(text)
+    index = 0
+    output = ""
+    for char in chars:
```

Comment:
You could use an enumeration here, rather than maintaining the separate `index` variable manually. However, if all the incorrect spaces are double spaces, you could just say `text.replace('  ', ' ')`